templates/create_keys.sh.j2

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# This script is taken from the official tmate-ssh-server repository.
+
+set -eu
+
+gen_key() {
+  keytype=$1
+  ks="${keytype}_"
+  key="{{ keys_dir }}/ssh_host_${ks}key"
+  if [ ! -e "${key}" ] ; then
+    ssh-keygen -t "${keytype}" -f "${key}" -N ''
+    echo ""
+  fi
+  SIG=$(ssh-keygen -l -E SHA256 -f "$key.pub" | cut -d ' ' -f 2)
+}
+
+mkdir -p keys
+gen_key rsa
+RSA_SIG=$SIG
+gen_key ed25519
+ED25519_SIG=$SIG
```

Comment:
From [previous PR](https://github.com/canonical/tmate-ssh-server-operator/pull/2)
[@jdkandersson] Do we need the `echo` here?